src/style.css

```diff
@@ -0,0 +1,64 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.head {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 40px 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.ul-items {
+  display: flex;
+  align-items: stretch;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+  padding: 0;
+  margin: 0 10px;
+  cursor: pointer;
+  height: 60px;
+  background: #fff;
+  line-height: 60px;
+
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+}
+
+.is-active::after {
+  content: "";
+  position: relative;
+  bottom: 4px;
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__menu {
+  height: 60px;
+}
```

Comment:
```suggestion
```
No need to specify height here, because your `.nav__item` has height 60px already